apps/backend/src/common/utils/stringHasher.ts

```diff
@@ -0,0 +1,23 @@
+import * as crypto from 'crypto';
+
+interface HashStringProps {
```

Comment:
@pawelroguski names are not best choices. I propose: 
`data`, `useSalt`, `usePepper`, `salt`
Because it is util function, it can be used in other places. It is easier to understand it this way.

Reply:
A little bit tomato, tomato... maybe for first three of them. The last one suggest that we have choice. Predefine salt or something else like auto generate new one salt in this case, by this function itself.

Reply:
I changed predefinedSalt to customSalt, that sounds even better than predefinedSalt

Reply:
basically every parameter is custom because it is util function. This is why i make point to change naming. Sugestion of choice is no needed in namings. It is defined in code with `?` so when someone else in future will use this function he will know which parameters are optional and which are not. He wont suggest that one from naming, because he will have `?` next to name.